config/recipes/traefik/README.asciidoc

```diff
@@ -0,0 +1,75 @@
+= Use Traefik to expose Elastic Stack applications
+
+This recipe illustrates how to use link:https://containo.us/traefik/[Traefik] to expose Elasticsearch, Kibana and APM Server resources to the outside world. It demonstrates how to route Elasticsearch traffic by node type and how to use Traefik middleware to add security and resiliency features.
+
+== Prerequisites
+
+- Traefik 2.1 or higher installed with the Kubernetes CRD provider. See: https://docs.traefik.io/providers/kubernetes-crd/
++
+NOTE: This recipe uses TLS to secure in-cluster communications. As the ECK generated certificates are self-signed, you will need to configure Traefik to trust those certificates. One way of doing this is to add the `--serverstransport.insecureskipverify=true` flag to the Traefik deployment.
```

Comment:
I guess having Traefik trust ECK self-signed CA makes things pretty hard to configure?

Reply:
Yes, it would be a complicated setup to configure it to trust specific certificates. Also, each cluster created by ECK has its own CA which makes it impractical if you have more than one cluster that you want to expose.